### ISSUE TITLE
Dark theme issues

### ISSUE THREAD

**Issue by KAMiKAZOW** [Nheko-Reborn/nheko#252]
Kinda the same as #118 but under Windows 10:

![image](https://user-images.githubusercontent.com/3259076/89118958-8b034e80-d4aa-11ea-84b4-4ca3606ab27d.png)

**Comment by redsky17** [comment]
I definitely think this is a duplicate of #118.  I'm going to close this in favor of that one.

**Comment by KAMiKAZOW** [comment]
How is a fixed Linux bug the same as an unfixed Windows bug? I opened both and I know for a fact that the Linux version was working fine after #125.
